src/screens/OtherProfile/index.js

```diff
@@ -387,11 +308,13 @@ const OtherProfile = () => {
     };
 
     const __renderMessageAndFollowButtonGroup = () => {
-      if (blockStatus.blocker) return <></>;
+      if (isBlocking) return <></>;
       return <React.Fragment>{__renderFollowingButton()}</React.Fragment>;
     };
 
-    if (blockStatus.blocked) return <></>;
+    console.log('isBlocked', isBlocked, 'isBlocking', isBlocking);
+
+    if (isBlocked) return <></>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variables `blockStatus.blocked` and `blockStatus.blocker` have been replaced with `isBlocked` and `isBlocking` respectively. Ensure that these new variables provide the same functionality as the old ones.



<!-- This is an auto-generated comment by OSS CodeRabbit -->